src/pages/Home/Home.page.jsx

```diff
@@ -1,47 +1,34 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
-import  Header  from '../../components/Header';
+import React, { useRef, useState, useContext } from 'react';
+import AppContext from '../../components/AppContext';
+import VideoView from '../../components/VideoView';
 import VideoCards from '../../components/VideoCards';
-
-import { useAuth } from '../../providers/Auth';
 import './Home.styles.css';
 
 
 function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
+  const [selectedVideo, setSelectedVideo] = useState('');
+  const myContext = useContext(AppContext);
```

Comment:
I saw you're using the `myContext.searchValue` to either use `selectedVideo` or the context one. What if you initialize your state with the context value itself?, this way you won't have to do that check down below.